packages/editor/src/store/selectors.js

```diff
@@ -495,18 +513,34 @@ export function isEditedPostEmpty( state ) {
 /**
  * Returns true if the post can be autosaved, or false otherwise.
  *
- * @param  {Object}  state Global application state.
+ * @param {Object} state    Global application state.
+ * @param {Object} autosave A raw autosave object from the REST API.
  *
  * @return {boolean} Whether the post can be autosaved.
  */
-export function isEditedPostAutosaveable( state ) {
+export const isEditedPostAutosaveable = createRegistrySelector( ( select ) => function( state, autosave ) {
```

Comment:
With being a registry selector now, is there any reason we should explicitly need to accept `autosave` as an argument now? What's wrong with leaving the implementation as if `arguments.length === 1` were the only option? It's not wrong for the editor to assume a post ID (and thus, select the autosave for that post ID).